components/calendly_v2/actions/create-scheduling-link/create-scheduling-link.mjs

```diff
@@ -0,0 +1,35 @@
+import calendly from "../../calendly_v2.app.mjs";
+
+export default {
+  key: "calendly-create-scheduling-link",
+  name: "Create a Scheduling Link",
+  description: "Creates a single-use scheduling link. [See the docs](https://calendly.stoplight.io/docs/api-docs/b3A6MzQyNTM0OQ-create-single-use-scheduling-link)",
+  version: "0.0.1",
+  type: "action",
+  props: {
+    calendly,
+    owner: {
+      propDefinition: [
+        calendly,
+        "eventType",
+      ],
+      label: "Owner",
+    },
+    maxEventCount: {
+      propDefinition: [
+        calendly,
+        "maxEventCount",
+      ],
+    },
+  },
+  async run({ $ }) {
+    let params = {};
+    params.max_event_count = this.maxEventCount;
```

Comment:
`params` can be a constant here because it's not being redefined, just added to.
```suggestion
  async run({ $ }) {
    const params = {};
    params.max_event_count = this.maxEventCount;
```

Reply:
Issue addressed